integrations/mistral/tests/test_mistral_chat_generator.py

```diff
@@ -267,3 +281,17 @@ def __call__(self, chunk: StreamingChunk) -> None:
 
         assert callback.counter > 1
         assert "Paris" in callback.responses
+
+    def test_live_run_with_tools(self, tools):
+        chat_messages = [ChatMessage.from_user("What's the weather like in Paris?")]
+        component = MistralChatGenerator(tools=tools)
+        results = component.run(chat_messages)
+        assert len(results["replies"]) == 1
+        message = results["replies"][0]
+
+        assert message.tool_calls
+        tool_call = message.tool_call
```

Comment:
we discussed about the removal of `assert not message.text`...

Which kind of text does the model produce along with a tool call?
Is it expected? I can't find it mentioned in [Mistral docs], but I haven't studied the topic in depth.